### PR TITLE
clarify titles of 7.4.3 bugs fixes

### PR DESCRIPTION
**Pull request by ctechro**

### CODE REVIEW

**Comment by tnguyen1** [comment]
ok to test

File: md/release-notes.md [hunk 1]
```diff
@@ -74,15 +74,15 @@ a new REST connector and important technology updates.
...
+* BS-16031	BDM.find() fails with Exception if element has a CHILD
 * BS-16106	bpm/archivedCase return error 500 instead of 404 if the case doesn't exist
 * BS-16145	WildFly bonita-start.bat failed if JAVA_HOME is set with a path including spaces
 * BS-16161	The user is not warned that the session will be closed when the BDM is deployed
 * BS-16170	JSON sorted results broken in widget Data table in Chrome
 * BS-16184	Tomcat does not test MySQL connection validity before giving it to Bonita application
 * BS-16206	Japanese chars in diagram name generate validation errors
-* BS-16214	Browse data (h2 console) new Studio Menu button not opening any browser
-* BS-16231	Connector definition - Widget input - document type (no contract input available)
+* BS-16214	The studio menu Development -> Business Data Model -> Browse Data (h2 console) doesn't open the bonded browser
```
Comment (tnguyen1) [inline md/release-notes.md:84]: What do you mean by "**bonded**" browser?

Comment (educhastenier) [reply md/release-notes.md:84]: "bound" maybe?

Comment (educhastenier) [reply md/release-notes.md:84]: BS-16231 has been removed ??!!

**Comment by dtranquang** [comment]
merge updates into #246

**Pull request closed without merging**